lms/static/js/commerce/views/receipt_view.js

```diff
@@ -61,6 +63,18 @@ var edx = edx || {};
                 $courseNamePlaceholder.text(responseData.name);
             });
         },
+        renderUserFullNamePlaceholder: function (username) {
+            var userModel = Backbone.Model.extend({
+              urlRoot: '/api/user/v1/accounts/',
+                url: function() {
+                    return this.urlRoot + this.id;
+                }
+            }), user;
+            user = new userModel({id:username});
```

Comment:
I would actually use ```this.user``` so the model can be accessed throughout the View (which will be the way it will be used after the refactor).